src/components/Dropdown.tsx

```diff
@@ -31,37 +38,57 @@ const Dropdown = ({
       setSelected(selected[0] === value ? [] : [value]);
     }
   };
+  const maxOptionLength = 15;
```

Comment:
for fixed constants, I recommend outside the component and using `const CAPITAL_SNAKE_CASE = `